packages/routing-utils/test/superstatic.spec.js

```diff
@@ -577,7 +642,18 @@ test('convertRewrites', () => {
     },
     {
       check: true,
-      dest: '/another/$1/$username/$pathname/$another/$host',
+      dest: '/another/$a/$b',
```

Comment:
Should this have a querystring with `?first=:first` appended?

Reply:
We don't automatically append the params from the source if at least one segment is being used in the `destination` currently so since the `has` segments are being used in the `destination` we don't auto append the `source` segment